Loyc-Slim.sln

```diff
@@ -0,0 +1,395 @@
+
```

Comment:
It was possible to avoid creating a second solution file by creating new solution configurations instead (e.g. Travis-Release.NET3, Travis-Release.NET4) but I guess it's okay to have two files.